spec/merch_calendar/retail_calendar_spec.rb

```diff
@@ -57,6 +60,35 @@
       expect(subject.end_of_quarter(2019, 4)).to eq Date.new(2020, 2, 1)
     end
   end
+  
+  describe "#quarter" do
+    it "returns the correct date" do
```

Comment:
This isn't correct.  `it "returns the correct quarter number"` would make more sense.

Reply:
👍